File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/test/sql/DefaultModelTest.java
package com.raizlabs.android.dbflow.test.sql;

import com.raizlabs.android.dbflow.sql.language.Delete;
import com.raizlabs.android.dbflow.test.FlowTestCase;

import org.junit.Test;

public class DefaultModelTest extends FlowTestCase {

    @Test
    public void testDefaultModel() {
        Delete.table(DefaultModel.class);
        DefaultModel defaultModel = new DefaultModel();
        defaultModel.setName("Test");
        defaultModel.save();
    }
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/test/sql/DefaultModelTest.kt
package com.raizlabs.android.dbflow.test.sql

import com.raizlabs.android.dbflow.sql.language.Delete
import com.raizlabs.android.dbflow.test.FlowTestCase

import org.junit.Test

class DefaultModelTest : FlowTestCase() {

    @Test
    fun testDefaultModel() {
        Delete.table(DefaultModel::class.java)
        val defaultModel = DefaultModel()
        defaultModel.name = "Test"
        defaultModel.save()
    }
}
